nomad/core_sched.go

```diff
@@ -494,7 +494,7 @@ OUTER:
 		}
 		var resp structs.NodeUpdateResponse
 		if err := c.srv.RPC("Node.Deregister", &req, &resp); err != nil {
-			c.logger.Error("node reap failed", "node_id", nodeID, err)
+			c.logger.Error("node reap failed", "node_id", nodeID, "error", err)
```

Comment:
Is there a quick way to ensure that logging functions get an odd number of arguments?  Maybe a linter?

Reply:
thanks to @evanphx, we do now! https://github.com/hashicorp/go-hclog/pull/31 (still need to add it to our make check)

Reply:
Yeah - it came in handy - with it, I found three more cases of such error in 7840ab51e2ca74f7ae5cd38933d1419900b72532!

Reply:
Also, planning to add it to our linters after https://github.com/hashicorp/go-hclog/pull/32 is merged.